types/ember__routing/test/route.ts

```diff
@@ -3,6 +3,7 @@ import Array from '@ember/array';
 import EmberObject from '@ember/object';
 import Controller from '@ember/controller';
 import Transition from '@ember/routing/-private/transition';
+import { assertType } from './lib/assert';
```

Comment:
I don't think this import is needed

Reply:
Fixed in the commit I'm about to push. Thanks!